style.css

```diff
@@ -0,0 +1,132 @@
+@media screen and (min-width: 375px) and (max-width: 767px) {
+  body {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-size: 12px;
+    font-family: 'Poppins', Arial, sans-serif;
+    background-color: #f7f7f9;
+  }
+
+  .main {
+    background-image: url(img/header-shapes.png);
+    background-size: cover;
+    border-radius: 0 0 10px 70px;
+    border: 1px solid #dfe1e6;
+    border-top: none;
+    width: auto;
+    height: 626px;
+  }
+
+  .tallbar {
+    height: 44px;
+    width: auto;
+    left: 0;
+    top: 0;
+    border-radius: 0;
+    background-color: transparent;
+  }
+
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    height: 44px;
+    width: 100%;
+    margin-top: 0;
+    border-radius: 0;
+    background-color: transparent;
+    position: fixed;
+  }
+
+  .logo {
+    color: #6070ff;
+    height: 20px;
+    width: auto;
+    margin-left: 24px;
+    margin-top: 10px;
+    font-family: 'Poppins', Arial, sans-serif;
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: center;
+  }
+
+  .fa-bars {
+    cursor: pointer;
+    color: #6070ff;
+    width: 16px;
+    margin-right: 36px;
+    margin-top: 7px;
+    font-size: 18px;
+    text-align: center;
+    height: 17px;
+  }
+
+  .title {
+    width: auto;
+    height: 150px;
+    margin-left: 24px;
+    margin-top: 134px;
+    margin-right: 24px;
+    font-family: 'Poppins', Arial, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    flex: none;
+  }
```

Comment:
- [ ] Good work styling your title here ✔️. However, I can see that you have some excess space between the logo and the title, please kindly ensure to remove this space in order to stick as much as possible to the Figma design. 🤓

    >Hint: try removing your `margin-top: 134px;` CSS property and use `padding-top: 134px;` instead. 🔧


This is the Figma design 👇🏽

![figma](https://user-images.githubusercontent.com/116701485/235274639-d8f6d297-04ac-41e3-a11d-6e07dc29b8ed.png)

This is your design 👇🏽(almost there 😅)

![design](https://user-images.githubusercontent.com/116701485/235274621-4991a0f2-5ba2-44eb-922a-2a8f5489e01e.png)

Reply:
- [ ] Also, please kindly ensure to reduce your title `font-size` a little bit to stick as much as possible to the Figma design. 🤓🚀